Libraries/Animated/NativeAnimatedHelper.js

```diff
@@ -550,7 +550,7 @@ function transformDataType(value: number | string): number | string {
   }
 }
 
-module.exports = {
+export default {
```

Comment:
So we're basically now exporting all of the methods in this file twice? Once as part of the default export, and once as their own property.

I don't think we should be doing that, and should choose one of these approaches (NativeAnimatedHelper is an internal API so we can make breaking changes if necessary)

Reply:
Oh, I wasn't sure if NativeAnimatedHelper was an internal API or not, but that should make things a lot simpler, gonna update imports so we always use `import NativeAnimatedHelper from '../NativeAnimatedHelper';`

Reply:
@javache done